### ISSUE TITLE
Better display of user's language

### ISSUE THREAD

**Issue by hakanai** [vrcx-team/VRCX#283]
Currently the user's language displays like this:

![image](https://user-images.githubusercontent.com/43236/138798613-5bf11a55-5fe3-40f9-a4b6-036e1a491028.png)

I'd rather like to avoid being associated with the United States of America so if anything I'd prefer to see a British flag here as at least that represents the origin of the language. Disclaimer: I'm not British either, so it would still be wrong, just better.

The 100% correct way to do this is just to show the names of the languages in the current user's language, until the world sorts this problem out for us and produces some useful icons to represent languages.

**Comment by GizmoOAO** [comment]
Maybe flags shouldn't be used to represent language.

[Flag as a symbol of language – stupidity or insult?](https://jkorpela.fi/flags.html)

**Comment by tschaerni** [comment]
I agree with Hakanai, the British flag for english would also be more consistent with everthing else, as websites also use the British flag for the english language. The US Flag would be good for ASL, as that is a Language signed in the US.

**Comment by SGA-max-faxalv** [comment]
Apathetic/Neutral to using the British flag to represent English. That is slightly more common than using the US flag (although that happens regularly as well). Another option is using the "splice", where US flag is top-left corner and British flag is other half, but that feels overly political correctness for a project like this.

I am however strongly against removing the flags altogether. They are an extremely useful visual tool, and is a worldwide industry-standard method for displaying languages. Almost every single language selection dialog features flag, users can easily associate flags with languages without having to read the text, and especially in dialog's such as the Friend List where all flags are shown in a list it would be a severe QoL regression if they were removed or replaced with text.

**Comment by hakanai** [comment]
If "industry-standard" is the way you say "non-standard", I completely agree. It's a UX standard not to use flags for languages.

**Comment by Foorack** [comment]
This issue strongly feels like it is driven purely for political motivations.

Whether it is the US or British flag used to represent English language does not matter to the average end-user. It is convenient using the US flag because the asset is already used when indicating instance region. But removing language flags altogether would be a severe feature-loss for the software.

**Comment by hakanai** [comment]
I disagree. We'd be losing nothing useful at all.

**Comment by Myrkie** [comment]
I disagree, we would be losing a bit of UX no point in removing them and as stated above its already used by instances to indicate region.

**Comment by hakanai** [comment]
Losing bad UX is called an improvement.

**Comment by DubyaDude** [comment]
![4a1](https://user-images.githubusercontent.com/13679909/185627532-80803292-3b99-43f0-91a5-7f6f707e55bf.jpg)

**Comment by CloudyBoop** [comment]
This is stupid

**Comment by Nekromateion** [comment]
Hey,
so I just fixed this entire problem with like 10 seconds of thinking.
You can thank me later.
![image](https://user-images.githubusercontent.com/43814053/185629365-70f0725c-60ba-45db-a0a0-5f722faeea80.png)

**Comment by CloudyBoop** [comment]
> Hey, so I just fixed this entire problem with like 10 seconds of thinking. You can thank me later. ![image](https://user-images.githubusercontent.com/43814053/185629365-70f0725c-60ba-45db-a0a0-5f722faeea80.png)

Amazing I approve 😭

**Comment by DubyaDude** [comment]
> Hey, so I just fixed this entire problem with like 10 seconds of thinking. You can thank me later. ![image](https://user-images.githubusercontent.com/43814053/185629365-70f0725c-60ba-45db-a0a0-5f722faeea80.png)

this is beautiful.
put it in prod

**Comment by OctoFloofy** [comment]
> If "industry-standard" is the way you say "non-standard", I completely agree. It's a UX standard not to use flags for languages.
> 
> "Almost every single language selection dialog features flag" is clearly false - I don't use any other services or apps which do this, and if I did, I would have filed the issue with them as well.
> 
> "users can easily associate flags with languages without having to read the text" - You should state which users you're talking about instead of making sweeping generalisations. In this particular case, you probably mean yourself only. Or maybe you're assuming everyone who speaks English lives in the US (i.e. you are American). Or maybe you live in one of those snowflake countries which only speaks a single language which also isn't used in any other country. Whatever your assumption here, it certainly doesn't make things easy for me. But hey, if they change it to the Australian flag, then it would make it easy for me. Is that what you're suggesting doing? :)

Mmh, i see Discord uses flags too 
![image](https://user-images.githubusercontent.com/33370769/185646701-0340d865-7d1a-4fa0-a57a-ba2e9b46a229.png)

**Comment by tschaerni** [comment]
Wikipedia also uses Flags:
https://en.wiktionary.org/wiki/Wiktionary:Language_flags_list

About the whole "don't use Flags, it's bad UX/UIX" debate, Picture this: sometimes the language is written in it's own language, and in such a case it's hard to find out which one is english, for example on asian websites. Flags are basically pictograms for languages. You don't need to be able to read any of the displayed language, but everyone can still figure out which Flag belongs to the language you can actually read.

So I would argue that flags are actually a better way of displaying Languages.

But I agree with a lot of other users in this thread, english should be the British flag, or atleast a British/US flag.
To be honest, we could argue that the US doesn't even have an official language at a federal level, so it should be disqualified for that.
Also the ASL flag is a US flag, which just brings confusion if you see two US flags if someone speaks english and can sign ASL.

**Comment by Myrkie** [comment]
> > Hey, so I just fixed this entire problem with like 10 seconds of thinking. You can thank me later. ![image](https://user-images.githubusercontent.com/43814053/185629365-70f0725c-60ba-45db-a0a0-5f722faeea80.png)
> 
> this is beautiful. put it in prod

agreed Lets get this merged

**Comment by hakanai** [comment]
> Mmh, i see Discord uses flags too

Discord at least lets you choose UK. It's too bad they don't have more options, but it's better than what VRChat gives us.

**Comment by LibanTheCanadian** [comment]
pretty late but in the files you can always change the US flag to the british flag (although if there's a US region sever it will become British)

![Image](https://github.com/user-attachments/assets/1114c80c-3612-4594-ba38-a185eb47644c)

**Comment by Sighyu** [comment]
off topic but @Myrkie add my new discord .limeboi
